PROTOCOL.md

```diff
@@ -1752,6 +1807,18 @@ The response contains multiple lines:
   - The lines are [files](#file) in the table (otherwise).
   - The ordering of the lines doesn't matter.
 
+When `responseformat=delta`, a sequence of JSON strings delimited by newline. Each line is a JSON object defined in [API Response Format in Delta](#api-response-format-in-delta).
+
+The response contains multiple lines:
+- The first line is [a JSON wrapper object](#json-wrapper-object-in-each-line-in-delta) containing the delta [Protocol](#protocol-in-delta-format) object.
+- The second line is [a JSON wrapper object](#json-wrapper-object-in-each-line-in-delta) containing the delta [Metadata](#metadata-in-delta-format) object.
+- The rest of the lines are [JSON wrapper objects](#json-wrapper-object-in-each-line-in-delta) for [Metadata](#metadata-in-delta-format), or [files](#file-in-delta-format).
+  - The lines are [files](#file-in-delta-format) which wraps the delta single action  in the table (otherwise), with possible historical [Metadata](#metadata-in-delta-format) (when startingVersion is set).
+  - The ordering of the lines doesn't matter.
+
+The delta actions are wrapped because they will be used to construct a local delta log on the recipient
+side and then leverage the delta library to read data. 
+
```

Comment:
Do you want to hold this off until we make the new changes discussed?

Reply:
updated the compatibility section with the latest discussions.